site/content/blogs/release-v118.en.md

```diff
@@ -24,9 +24,95 @@ See the section [Overview](../docs/concepts/overview.en.md) for a more detailed
 _Contributed by [Efe Karakus](https://github.com/efekarakus/)_
 
+Copilot provides the `copilot pipeline` commands to create continuous delivery pipelines to automatically release microservices in your git repository.  
+Prior to v1.18, all services and jobs defined in your git repository got deployed in parallel for each stage.
+For example, given a monorepo with three microservices: `frontend`, `orders`, `warehouse`. All of them got deployed at the same time
+to the `test` and `prod` environments:
+
+=== "Pipeline"
+    ![Rendered pipeline](../assets/images/pipeline-default.png)  
+=== "Pipeline Manifest"
+
+    ```yaml
+    name: release
+    source:
+      provider: GitHub
+      properties:
+        branch: main
+        repository: https://github.com/user/repo
+    stages:
+    - name: test
+    - name: prod
+      requires_approval: true
+    ```
+=== "Repository Layout"
+
+    ```
+    copilot
+    ├── frontend
+    │   └── manifest.yml
+    ├── orders
+    │   └── manifest.yml
+    └── warehouse
+        └── manifest.yml
+    ```
+
+Starting with v1.18, you can control the order of your deployments in your pipeline with the new [`deployments` field](../docs/manifest/pipeline.en.md#stages-deployments).  
+```yaml
+stages:
+  - name: test
+    deployments:
+      orders:
+      warehouse:
+      frontend:
+        depends_on: [orders, warehouse]
+  - name: prod
+    require_approval: true
+    deployments:
+      orders:
+      warehouse:
+      frontend:
+        depends_on: [orders, warehouse]
+```
+
+With the manifest above, we're declaring that the `orders` and `warehouse` services should be deployed prior to the `frontend` so that clients can't send new API requests
+before the downstream services are ready to accept them. Copilot figures out in which order the stacks should be deployed, and the resulting CodePipeline looks as follows:
+
+![Rendered ordered pipeline](../assets/images/pipeline-ordered.png)
+
+There are a few other enhancements that come with the new `deployments` field:
+
+1. It is now possible for monorepos to configure which services or jobs to deploy in a pipeline. For example, I can limit 
+   the pipeline to only deploy the `orders` microservice:
+   ```yaml
+   deployments:
+     orders:
+   ```
+2. Your pipelines can now deploy standalone CloudFormation templates that are not generated by Copilot. For example, if we have a repository structured as follows:
+   ```
+   copilot
+    ├── api
+    │   └── manifest.yml
+    └── templates
+        ├── cognito.params.json
+        └── cognito.yml
+   ```
+   Then I can leverage the new `stack_name`, `template_path` and `template_config` fields under deployments to specify deploying the cognito cloudformation stack in my pipeline:
+   ```yaml
+   deployments:
+     cognito:
+       stack_name: myapp-test-cognito
+       template_path: infrastructure/templates/cognito.yml
+       template_config: infrastructure/templates/cognito.params.json
```

Comment:
A random question not related to the doc: does the location of `template_path` and `template_config` have to be in `infrastructure/templates/*`. I wonder if we should make it default for users, so that we can automate:

```yaml
   deployments:
     cognito:
       stack_name: myapp-test-cognito
       template_path: cognito.yml
       template_config: cognito.params.json
```

Reply:
that'd be cool unfortunately they have to be at least under `infrastructure` because of our buildspec:
```yaml
# in our buildspec.yml file
artifacts:
  files:
    - "infrastructure/*"
```

I could just copy under `infrastructure` instead to make the example shorter though 👍